src/main/java/creativei/helper/ResponseHelper.java

```diff
@@ -65,8 +65,8 @@ private static AddressVo getCreateAddressResponseData(InquiryAddress inquiryAddr
     }
 
     private static List<EducationVo> getCreateEducationResponseData(List<InquiryEducation> inquiryEducation, List<EducationVo> educationVos) {
-        EducationVo educationVo=new EducationVo();
         for(InquiryEducation inquiryEducation1:inquiryEducation){
```

Comment:
List name should be plural. Use proper naming conventions

Reply:
issue solved in commit 6ffb6af